Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorInstance.cpp

```diff
@@ -632,6 +632,7 @@ namespace AZ::Render
         if (m_meshHandle)
         {
             m_meshFeatureProcessor->ReleaseMesh(*m_meshHandle);
+            MeshHandleStateNotificationBus::Event(m_entityId, &MeshHandleStateNotificationBus::Events::OnMeshHandleSet, &(*m_meshHandle));
```

Comment:
Do you also need to send this event when the mesh handle is acquired?

Reply:
@jonawals ^